packages/app/src/cli/services/dev/extension/bundler.ts

```diff
@@ -23,17 +23,22 @@ export async function setupBundlerAndFileWatcher(options: FileWatcherOptions) {
 
   const bundlers: Promise<void>[] = []
 
-  options.devOptions.extensions.forEach((extension) => {
+  // eslint-disable-next-line @typescript-eslint/no-misused-promises
+  options.devOptions.extensions.forEach(async (extension) => {
```

Comment:
Super late to this, but was there a reason we needed the `async` and the explicit eslint override?